starlark/src/stdlib/mod.rs

```diff
@@ -717,10 +729,19 @@ starlark_module! {global_functions =>
     /// list(range(10, 3, -2))                  == [10, 8, 6, 4]
     /// # )"#).unwrap());
     /// ```
-    range(#a1, #a2 = None, #a3 = None) {
-        let start = if a2.get_type() == "NoneType" { 0 } else { a1.to_int()? };
-        let stop = if a2.get_type() == "NoneType" { a1.to_int()? } else { a2.to_int()? };
-        let step = if a3.get_type() == "NoneType" { 1 } else { a3.to_int()? };
+    range(#a1, ?#a2, ?#a3) {
```

Comment:
The range case is not clear to me whether we should does not consider "None" as being acceptable. Anyway I am fine going either way.